packages/workbox-cli/src/lib/help-text.js

```diff
@@ -18,9 +18,13 @@ module.exports = `Usage:
 $ workbox <command> [options]
 
 Commands:
-  wizard
+  wizard [--swSrc=<path/to/service-worker.js>]
```

Comment:
A lot of tools opt for `<name> init` or `<name> create` to designate the action you're about to perform. Is there a reason we felt `wizard` made more sense here?